governance/proposals/010-arbitrum-l1-l2-messaging.js

```diff
@@ -0,0 +1,225 @@
+const ethers = require('ethers')
+const {
+  L1ToL2MessageGasEstimator,
+} = require('@arbitrum/sdk/dist/lib/message/L1ToL2MessageGasEstimator')
+const { arbLog } = require('arb-shared-dependencies')
+const {
+  EthBridger,
+  getL2Network,
+  addDefaultLocalNetwork,
+} = require('@arbitrum/sdk')
+const { getBaseFee } = require('@arbitrum/sdk/dist/lib/utils/lib')
+
+const ERC20_ABI = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'account',
+        type: 'address',
+      },
+    ],
+    name: 'balanceOf',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256',
+      },
+    ],
+    stateMutability: 'view',
+    type: 'function',
+  },
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'to',
+        type: 'address',
+      },
+      {
+        internalType: 'uint256',
+        name: 'amount',
+        type: 'uint256',
+      },
+    ],
+    name: 'transfer',
+    outputs: [
+      {
+        internalType: 'bool',
+        name: '',
+        type: 'bool',
+      },
+    ],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+]
+const INBOX_ABI = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'to',
+        type: 'address',
+      },
+      {
+        internalType: 'uint256',
+        name: 'l2CallValue',
+        type: 'uint256',
+      },
+      {
+        internalType: 'uint256',
+        name: 'maxSubmissionCost',
+        type: 'uint256',
+      },
+      {
+        internalType: 'address',
+        name: 'excessFeeRefundAddress',
+        type: 'address',
+      },
+      {
+        internalType: 'address',
+        name: 'callValueRefundAddress',
+        type: 'address',
+      },
+      {
+        internalType: 'uint256',
+        name: 'gasLimit',
+        type: 'uint256',
+      },
+      {
+        internalType: 'uint256',
+        name: 'maxFeePerGas',
+        type: 'uint256',
+      },
+      {
+        internalType: 'bytes',
+        name: 'data',
+        type: 'bytes',
+      },
+    ],
+    name: 'createRetryableTicket',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256',
+      },
+    ],
+    stateMutability: 'payable',
+    type: 'function',
+  },
+]
+
+/**
+ * Set up: instantiate L1 / L2 wallets connected to providers
+ */
+const walletPrivateKey = 'Your_Private_key'
+const L1RPC = 'https://mainnet.infura.io/v3/<your_infura_api_key>'
+const L2RPC = 'https://arbitrum-mainnet.infura.io/v3/<your_infura_api_key>'
+const l1Provider = new ethers.JsonRpcProvider(L1RPC)
+const l2Provider = new ethers.JsonRpcProvider(L2RPC)
+// const l1Wallet = new ethers.Wallet(walletPrivateKey, l1Provider)
+const l2Wallet = new ethers.Wallet(walletPrivateKey, l2Provider)
+
+module.exports = async () => {
+  await arbLog('Cross-chain Proposer')
```

Comment:
what is `arbLog` used for ?

Reply:
It came with the arbitrum sdk, its more or less a branding/description of the script.

Reply:
Can we just use a regular `console.log` please?